bucket/poetry.json

```
@@ -9,10 +9,10 @@
     "installer": {
         "script": [
             "$env:POETRY_HOME=\"$dir\"",
-            "python3 \"$dir\\$fname\" --version $version"
+            "Invoke-Expression \"$(scoop which python) `\"$dir\\$fname`\" --version $version\""
         ]
     },
-    "bin": "venv\\Scripts\\poetry.exe",
+    "bin": "bin\\poetry.exe",
```

```suggestion
    "bin": "venv\\Scripts\\poetry.exe",
```
The `bin\\poetry.exe` file is a symlink. It's better to directly shim `venv\\Scripts\\poetry.exe`

---

the `bin\\poetry.exe` ' s size is 104 KB, maybe there are some other logics, and I think maybe bin is more readable than `venv\\scripts`.
Also, as the poetry installation says, the `bin` should be added to the `PATH` variable, If we use the `bin` field instead, I think it should be set to `bin\\poetry.exe`

---

Okay